macros/contexts/contextInequalitiesAllowStrings.pl

```diff
@@ -14,8 +14,8 @@ =head3 Allows string answers for the set of real numbers and the empty set
 
 =pod
```

Comment:
You should also deleted the `=pod` here.  That doesn't do anything after `=head2`.

Reply:
I also see an `=cut` statement immediately before this on line 11.  That is also unneeded.

Reply:
I ran `podchecker` on this and it says the `=head3` should be a `=head2` (I am paraphrasing).  Also the parts in `=head1 NAME` and `=head1 Synopsis` should have no indentation.  `podchecker` says "*** WARNING: Verbatim paragraph in NAME section" on this.